src/Controls/StateControl.js

```diff
@@ -27,13 +41,178 @@ const CONTROL_KEYS = {
  * @class StateControl
  *
  */
-class StateControl {
-    constructor() {
+class StateControl extends THREE.EventDispatcher {
+    constructor(eventTarget, options = {}) {
```

Comment:
rename by dom element

Reply:
I renamed the property as well as the parameter to `domElement`.